dallinger/experiment_server/experiment_server.py

```diff
@@ -747,21 +747,22 @@ def create_participant(worker_id, hit_id, assignment_id, mode):
     # Ping back to the recruiter that one of their participants has joined:
     recruiters.for_experiment(exp).notify_recruited(participant)
```

Comment:
Tiny thing, but worth it I think, to reduce the mental load:
```
+    recruiter = recruiters.for_experiment(exp)

     # Ping back to the recruiter that one of their participants has joined:
-    recruiters.for_experiment(exp).notify_recruited(participant)
+    recruiter.notify_recruited(participant)

     overrecruited = exp.is_overrecruited(waiting_count)
     if not overrecruited:
         # We either had no quorum or we have not overrecruited, inform the
         # recruiter that this participant will be seeing the experiment
-        recruiters.for_experiment(exp).notify_using(participant)
+        recruiter.notify_using(participant)
```

Reply:
Good call